src/components/results/result.vue

```diff
@@ -11,22 +11,31 @@
       </BaseResultImage>
     </BaseResultLink>
 
-    <div class="x-result__overlay x-list x-list--horizontal">
-      <BaseAddToCart :result="result" class="x-list__item--expand">
-        <CartIcon />
+    <div v-if="$x.device === 'desktop'" class="x-result__overlay x-list x-list--horizontal">
+      <BaseAddToCart
+        :result="result"
+        class="x-list__item--expand x-button--pill x-button--tertiary x-margin--05"
+      >
         <span>{{ $t('result.addToCart') }}</span>
       </BaseAddToCart>
     </div>
 
-    <BaseResultLink class="x-result__description x-list x-list--gap-04" :result="result">
-      <div class="x-list x-list--gap-02">
-        <h1 class="x-small x-text--bold">{{ result.season }}</h1>
-        <h2 class="x-text x-text--secondary x-ellipsis" data-test="result-title">
-          {{ result.name }}
-        </h2>
+    <BaseResultLink
+      class="x-result__description x-list"
```

Comment:
```suggestion
      class="x-result__description"
```

With this change you fix the problem with the topclicked styles.

Reply:
But if I delete this class then the gap is not applied 🤔 
Got it working by adding the `x-list--vertical` class.